carbon-design-system/carbon
# [VITE]: Upgrade incremental-migration to use vite

Closes #14462 

Updated the `examples/incremental-migration` to use vite instead of nextjs.

The example was getting an error before. When running with nexjs inside the monorepo it gets an error, but if we run the example outside of the monorepo it is working fine.

Also it is a good exercise to unserstand what to change when we start to migrating the CRA / NextJS examples to Vite in this [issue](https://github.com/carbon-design-system/carbon/issues/13637).

#### Changelog

**New**

- `examples/incremental-migration` using vite

#### Testing / Reviewing

- Pull down locally
- Go to the `examples/incremental-migration-vite` folder and run `yarn && yarn dev`
- Project should start

It is taking a little bit of time to first start. The same thing used to happen with the NextJS example. 
I'm still trying to understand that odd behavior. 🤔 

_**Before merge I want to get this PR up: https://github.com/carbon-design-system/carbon/pull/14219**_
